src/sync/queries.ts

```diff
@@ -19,40 +38,80 @@ export const getPullRequests = `query ($owner: String!, $repo: String!, $per_pag
             headRef {
               name
             }
-            number
-            state
-            title
+            comments {
+              totalCount
+            }
+          }
+        }
+      }
+    }
+  }`;
+
+export const getPullRequestReviews = `query($owner: String!, $repo: String!, $pullRequestNumber: Int!, $cursor: String) {
+  repository(owner: $owner, name: $repo) {
+    pullRequest(number: $pullRequestNumber) {
+      reviews(first: 100, after: $cursor) {
+        pageInfo {
+          hasNextPage
+          startCursor
+          endCursor
+        }
+        totalCount
+        edges {
+          cursor
+          node {
             body
-            updatedAt
+            state
             url
+            submittedAt
+            author {
+              ... on User {
+                avatarUrl
+                login
+                email
+                url
+                name
+              }
+            }
           }
         }
       }
     }
-  }`;
+  }
+}`;
 
-export const getCommits = (includeChangedFiles?: boolean) => `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String, $default_ref: String!) {
+export const getCommits = `query ($owner: String!, $repo: String!, $cursor: String, $default_ref: String!) {
     repository(owner: $owner, name: $repo){
       ref(qualifiedName: $default_ref) {
         target {
           ... on Commit {
-            history(first: $per_page, after: $cursor) {
+            history(first: 100, after: $cursor) {
+              pageInfo {
+                hasNextPage
+                startCursor
+                endCursor
+              }
+              totalCount
               edges {
                 cursor
                 node {
-                  author {
-                    avatarUrl
-                    email
-                    name
-                    user {
-                      url
-                    }
-                  }
-                  authoredDate
-                  message
-                  oid
-                  url
-                  ${includeChangedFiles ? "changedFiles" : ""}
+                  abbreviatedOid
+					        oid
+					        message
+					        authoredDate
+					        url
+					        changedFiles
```

Comment:
This field causes GitHub to return errors. What happened to the logic I built to retry without this field in a certain error (behind the feature flag `RETRY_WITHOUT_CHANGED_FILES`)? There must be a badly resolved merge conflict here.

Reply:
If Github returns an error on a field, which should be optional, this is a Github issue that needs to be addressed.  But we can have workaround because even if the field query does error, it still returns the data up to that point, so we should be able to continue after that point on error.

The other possibility is to not query `changedFiles` until this is fixed on the github side.